test/common/common/BUILD

```diff
@@ -60,6 +60,8 @@ envoy_cc_fuzz_test(
     name = "hash_fuzz_test",
     srcs = ["hash_fuzz_test.cc"],
     corpus = "hash_corpus",
+    # Fuzzer is stable, no bugs, simple test target; avoid emitting CO2.
+    tags = ["no_fuzz"],
```

Comment:
Is there a way we could have a reduced fuzz for these? It would be nice to catch regressions, but to force them to back-off to ever N days or weeks to avoid burning CPU bandwidth unnecessarily.

Reply:
Originally I thought to rely on internal equivalents for these, but I think if there's a way to rebuild them only once every N days, that would be better. Send an email about increasing memory usage or some workaround like this.

Reply:
I would think for these simple dependencies it would be nice to force fuzz them for N weeks after a dependency bump too, but that may be too complicated to automate the build tags after detecting a bump.